index.android.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-// import './src/ReactotronConfig';
+import './src/ReactotronConfig';
```

Comment:
Include Reactotron into our execution flow and it must be done at first line as per docs.

Reply:
I suspect the start of `ZulipMobile.js` is good enough and you do not need to repeat it in two files.

Reply:
Ok I will do that.